tests/subsys/logging/log_timestamp/testcase.yaml

```diff
@@ -0,0 +1,9 @@
+common:
+  integration_platforms:
+    - native_posix
+
+tests:
+  logging.log_output_custom_timestamp:
```

Comment:
since test could also be run in configuration without custom timestamp function add:
```
logging.log_output_default_timestamp:
    tags: log_output logging timestamp
```

Reply:
Added.